sphinx6-upgrade.rst

```diff
@@ -0,0 +1,56 @@
+.. post:: January 4th, 2022
+   :tags: newsletter, python
```

Comment:
This isn't a newsletter, though we do need to publish that as well soon :)

Reply:
Oh yes, that was in a blind spot, thanks!

Reply:
Fixed